scripts/bundle-config-rules.ts

```diff
@@ -0,0 +1,17 @@
+import fs from 'node:fs';
```

Comment:
Did you consider putting this in the config repo instead? It might be easier if the automation runs in that repo, so it can create a PR to the config, and maybe reuse the schemas from there.